xblocks_contrib/annotatable/annotatable.py

```diff
@@ -0,0 +1,115 @@
+"""TO-DO: Write a description of what this XBlock is."""
+
+import pkg_resources
```

Comment:
We should remove it in real implementation of `Annotatble XBlock` as it is deprecated.
Relevant PR: https://github.com/openedx/XBlock/pull/716